symengine/subs.h

```diff
@@ -252,8 +363,8 @@ class SubsVisitor : public BaseVisitor<SubsVisitor>
                         break;
                     }
                 } else {
-                    result_
-                        = make_rcp<const Subs>(x.rcp_from_this(), subs_dict_);
+                    result_ = make_rcp<const Subs>(x.rcp_from_this(),
```

Comment:
@isuruf, If `Derivative` is moved to `xreplace`,  does returning an instance of `Subs` class on a call to `xreplace` make sense?